src/Parser/Core/Modules/Items/ArchimondesHatredReborn.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import ITEMS from 'common/ITEMS';
+
+import Module from 'Parser/Core/Module';
+
+class ArchimondesHatredReborn extends Module {
+  healing = 0;
+
+  on_initialized() {
+    if (!this.owner.error) {
+      this.active = this.owner.selectedCombatant.hasTrinket(ITEMS.ARCHIMONDES_HATRED_REBORN.id);
+    }
+  }
+
+  on_byPlayer_absorbed(event) {
+    const spellId = event.ability.guid;
+    if (spellId === ITEMS.ARCHIMONDES_HATRED_REBORN.absorbId) {
+      this.healing += event.amount;
+    }
+  }
+
+  item() {
+    return {
+      item: ITEMS.ARCHIMONDES_HATRED_REBORN,
+      result: (
+        <dfn data-tip={`The total damaged absorbed by Archimonde's Hatred Reborn was ${this.owner.formatItemAbsorbDone(this.healing)}.`} >
```

Comment:
No I don't really mind the tooltips, just looking for ways to keep them to a minimum.

How are you using absolute numbers when analyzing logs? If you're comparing to other tanks I reckon you should look at DTPS / HPS instead since this is more likely to be comparable even when fight times are different. The nice thing about the *% of total / HPS values* is that they give you two numbers that should be similar across all logs.

For healers Prydaz will generally be stuck at around 33k HPS, or 5% of total healing done. Across all fights you'll find this to be very similar. The other decent legendaries are also worth around 5% of total healing done, so the comparison becomes pretty simple that if your legendary is worth less than 5% you may need to look at something else (this is a simplified PoV).

The downside of the percentage display is that it depends on your total throughput, if that's shit something like Prydaz will appear to perform better than something that scales with your total healing (and worse if you have a lot of throughput). So that's why the HPS value is there; this does not inflate when your total throughput is lower so can be used more reliably, but is harder to work with.